### PR TITLE
add support for docker-compose's `dockerfile` config option

### PR DESCRIPTION
**Pull request by gmelika**
This is to add support for the `dockerfile` config option as described here: https://docs.docker.com/compose/compose-file/#dockerfile

### CODE REVIEW

**Comment by gmelika** [comment]
@nzoschke I've fixed the issue with Dockerfile. please test with this and let me know

File: api/manifest/manifest.go [hunk 1]
```diff
@@ -153,6 +154,14 @@ func buildSync(source, tag string, cache bool) error {
 		args = append(args, "--no-cache")
 	}
 
+	if dockerfile != "" {
+		args = append(args, "-f")
```
Comment (ddollar) [inline api/manifest/manifest.go]: Could simplify this a bit:
`args = append(args, "-f", "/" + dockerfile))`

Comment (gmelika) [inline api/manifest/manifest.go]: was going for speed with Buffer.WriteString instead, but I guess that's an overkill during the build step.  I'll change it

**Comment by gmelika** [comment]
good call.  rebased and pushed the change

**Comment by ddollar** [comment]
You can try this with `convox rack update 20160128212241-support-dockerfile`

**Comment by ddollar** [comment]
Thanks!

**Pull request merged**